server/database/queries/users/addUserInterests.js

```diff
@@ -0,0 +1,14 @@
+const dbConnection = require('../../config/dbConnection');
+
+const interestsQueryMaker = (interests) => {
+  const query = 'INSERT INTO user_interest(interest_id, user_id) VALUES ($1,$2)';
+  const rest = interests.slice(1).map((e, i) => `, ($1,$${i + 3})`).join('');
+  return query + rest;
+};
+exports.addUserInterests = (interestsIds, userId) => {
+  const sql = {
+    text: interestsQueryMaker(interestsIds),
+    values: [...interestsIds, userId],
```

Comment:
according to the previous change you should make changes here also 
```suggestion
    values: [userId, ...interestsIds],
```